CHANGELOG.md

```diff
@@ -30,7 +30,7 @@ The release also includes several bug fixes. Telemetry reports now include new a
 **Bugfixes**
 * #3808 Properly handle `max_retries` option
 * #3863 Fix remote transaction heal logic
-* #3869 Fix ALTER SET/DROP NULL contstraint on distributed hypertable
+* #3869 Fix ALTER SET/DROP NULL constraint on distributed hypertable
```

Comment:
Some projects don't like to change changelogs, let me know if this is one of them. I'm happy to exclude files.